begin.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Apple solo project</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" type="text/css" href="reset.css" />
+    <link rel="stylesheet" type="text/css" href="begin.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <style>
+      A {
+        text-decoration: none;
+      }
+    </style>
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+  </head>
+  <body>
+    <main>
+      <nav>
+        <div class="nav-container">
+          <a href="#" class="fab fa-apple"></a>
+          <a href="#">Store</a>
+          <a href="#">Mac</a>
+          <a href="#">iPod</a>
+          <a href="#">iPhone</a>
+          <a href="#">iPad</a>
+          <a href="#">iTunes</a>
+          <a href="#">Support</a>
+          <div><input type="text" /></div>
+        </div>
+      </nav>
+
+      <section class="main-body">
+        <div class="links-container">
+          <a
+            href="https://web.archive.org/web/20140228175622/http://www.apple.com/your-verse/"
+            target="_blank"
+            ><img
+              src="assets/your_verse_title_small.png"
+              alt="What will your verse be?"
+            />
+          </a>
+          <a
+            href="https://web.archive.org/web/20140228175622/http://www.apple.com/your-verse/"
+            target="_blank"
+            ><img
+              src="assets/your_verse_subhead_small.png"
+              alt="Everyone's using iPad"
+            />
+          </a>
+          <a
+            class="sub-watch"
+            href="https://web.archive.org/web/20140228175622/http://www.apple.com/your-verse/"
+            target="_blank"
+            ><h3>
+              Watch the "Your Verse" film and explore the stories<i
+                class="far fa-play-circle"
+              ></i>
+            </h3>
+          </a>
+        </div>
+      </section>
+    </main>
+
+    <section class="bottom-side">
+      <div class="bottom-imgs">
+        <a
+          class="img-bottom"
+          href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone-5s/"
+          target="_blank"
+          ><img src="assets/img1.jpg" alt="iPhone5s"
+        /></a>
```

Comment:
[REQUIRED] I agree with you that there is no header to add but we have to fix all the HTML validator error. It will help in SEO and is always a good practice to do so. We can use div tag instead of section.

Reply:
Done